books/management/commands/resync.py

```diff
@@ -71,7 +71,7 @@ def handle(self, *args, **options):
             if book:
                 if success:
                     counter['success'] = counter['success'] + 1
-                    self.stdout.write(self.style.SUCCESS(
+                    self.stdout.write(self.style.HTTP_REDIRECT(
```

Comment:
Certainly, the pull request is great as is and any refinements can wait until whenever we get around to it later.